### DISCUSSION TITLE
6.12 二分图最大匹配

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#159] [Comments]
# 6.12 二分图最大匹配

[https://algo.itcharge.cn/06_graph/06_12_graph_bipartite_matching/](https://algo.itcharge.cn/06_graph/06_12_graph_bipartite_matching/)

**Comment by akxuan** [comment]
这里为什么没有东西呀

**Reply by itcharge** [reply]
您好，文章已更新，欢迎阅读
